data/lhc-hxswg/makeSMtables.py

```
@@ -213,11 +272,20 @@ def formatval(v):
+    v = round(v, 4)
     return "{:+}".format(v)
 
 
 def main(o):
-    f = xlrd.open_workbook(o.input)
+    try:
+        if o.input.endswith(".xls"):
+            f = xlrd.open_workbook(o.input)
+        else:
+            f = load_workbook(o.input)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)